target/linux/kirkwood/base-files/etc/board.d/02_network

```diff
@@ -26,8 +26,8 @@ case "$board" in
 	;;
 "linksys,audi"|\
 "linksys,viper")
-	ucidef_add_switch "switch0" \
-		"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "5@eth0" "6@eth1"
+	ucidef_set_interfaces_lan_wan "ethernet1 ethernet2 ethernet3 ethernet4" "internet"
+	ucidef_set_interface_macaddr "wan" $( mtd_get_mac_ascii u_env eth1addr )
```

Comment:
Where is that coming from?

Reply:
In swconfig wan mac is configured to eth1. In dsa configuration, wan mac is inherited from eth0 and is the same as lan addresses.

Reply:
> In swconfig wan mac is configured to eth1

Yes, but where did it get its MAC address from back then?

Reply:
In u-Boot. OpenWrt doesn't touched MAC addresses before patch.

Reply:
Okay, then this seems legitimate.
Please add a short comment to the commit message, to prevent other people from asking the same question, with the following information:
old: eth1 address initialized from u-boot
now: wan in switch, so add address manually
:-)

Reply:
Done.

Reply:
@CHKDSK88 Please move the comment to the _commit message_. No need to consume space on the device, it's just for people looking that up in git history.

Reply:
Thank You. Done.